### PR TITLE
Add vaos relationships endpoint

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Summary

- *This work is behind a feature toggle (flipper): NO*
- Added a new `/relationships` endpoint to the VAOS module that will be used for future appointment booking work. This uses VPG to pull patient provider relationship information from Oracle Health and VAOS

## Related issue(s)

- https://app.zenhub.com/workspaces/appointments-oracle-health-integration-65a6e99ea522640e4d09393b/issues/gh/department-of-veterans-affairs/va.gov-team/90606

## Testing done

- [x] *New code is covered by unit tests*

## What areas of the site does it impact?
This will have no impact on the site.

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by drewconnelly** [comment]
From the Summary:
> This uses VPG to pull Oracle Health patient relationship information

The VPG relationships endpoint actually consolidates relationship info from both Oracle Health _and_ VAOS, so we should clarify that out here.

File: modules/vaos/app/docs/vaos/v2/vaos_v2.yaml [hunk 1]
```diff
@@ -1245,6 +1306,29 @@ components:
...
+          type: object
+          properties:
+            provider:
+              $ref: "#/components/schemas/PractitionerReference"
+            location:
+              $ref: "#/components/schemas/LocationReference"
+            clinic:
+              $ref: "#/components/schemas/ClinicReference"
+            serviceType:
+              $ref: '#/components/schemas/CodeableConcept'
```
Comment (drewconnelly) [inline modules/vaos/app/docs/vaos/v2/vaos_v2.yaml:1327]: Let's change this to double quotes to be consistent.

**Review by drewconnelly — APPROVED** [review]
A couple of minor comments, but otherwise LGTM.

**Review by drewconnelly — APPROVED** [review]
Approved.

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**